app/models/site_notice.rb

```diff
@@ -43,7 +43,9 @@ def document
   def preview_content
     start_deadline unless consultation_started?
 
-    I18n.t("site_notice_template",
+    site_notice_type = consultation.planning_application.environment_impact_assessment&.required? ? "eia" : "general"
```

Comment:
It's a very minor point, but wondering if "default" might be better than "general" here, especially if we have to add more types of site notice later?